index.html

```diff
@@ -315,6 +315,55 @@ <h3>
             </div>
         </div>
     </section>
+    <!-- form section -->
+    <section class="form-section">
+        <article class="form-article">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </article>
+        <form novalidate action="https://formspree.io/f/xqkjrlry" method="post">
+
+            <input type="text" id="full-name" class="form-controll" maxlength="30" name="full_name"
+                placeholder="First name" required>
+            <input type="email" id="email" class="form-controll" name="user-email" placeholder="Email address"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required />
+            <textarea name="message" maxlength="500" placeholder="Enter message here" required></textarea>
+            <button class="form_button" type="submit">Get in touch</button>
+        </form>
+    </section>
+    <hr>
+    <footer>
+        <div class="icons">
+            <ul>
+                <li>
+                    <a href="#">
+                        <img src="./images/Disabled.png" alt="github" /></a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Disabled-1.png" alt="Linkedin" />
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Enabled.png" alt="Angelist" />
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Disabled-2.png" alt="Twitter" />
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Disabled-3.png" alt="Medium" />
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="shape">
+
+        </div>
+    </footer>
```

Comment:
- [ ] Great work implementing the changes suggested by the first reviewer. However, the change you made still needs to be improved. The icons that you implemented now are tiny compared to that of the Figma template. I would advise you to make the `height` and `width` of the icons 30px` each since you implemented them as images. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/206298766-fb2c7b48-7ac2-4cdc-8aa0-202103be19db.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/206298854-4338e442-c862-43fb-a350-f2677b595fc0.png)